module/VuFind/config/module.config.php

```diff
@@ -338,7 +338,7 @@
     'service_manager' => [
         'allow_override' => true,
         'factories' => [
-            'League\CommonMark\MarkdownConverterInterface' => 'VuFind\Service\MarkdownFactory',
+            'League\CommonMark\ConverterInterface' => 'VuFind\Service\MarkdownFactory',
```

Comment:
Should we create an alias for the old service name for backward compatibility, or just make a note in the changelog?

Reply:
This does make sense. I've added an alias.